app/src/main/java/cm/aptoide/pt/network/data/PreferencesPersister.kt

```diff
@@ -0,0 +1,24 @@
+package cm.aptoide.pt.network.data
+
+import androidx.datastore.core.DataStore
+import androidx.datastore.preferences.core.Preferences
+import androidx.datastore.preferences.core.edit
+import androidx.datastore.preferences.core.stringPreferencesKey
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class PreferencesPersister @Inject constructor(private val datastore: DataStore<Preferences>) {
```

Comment:
like i said before i would give it a different name to describe better what it is persisting

Reply:
I've change its name, but check it again to see if it should be more descriptive